src/Employer/Employer.Web/Views/VacancyPreview/AdvertPreview.cshtml

```diff
@@ -10,22 +10,32 @@
   <a asp-route="@RouteNames.EmployerCheckYourAnswersGet" asp-all-route-data="@Model.RouteDictionary" esfa-automation="link-back" class="govuk-back-link">Back to check your answers</a>
 }
 
-<h1 class="govuk-heading-xl">Preview your advert</h1>
+<div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-xl">Preview your advert</h1>
+    <div class="govuk-inset-text">
+        Proofread your advert before submitting. We might reject your advert if it has spelling or formatting errors.
+    </div>
+</div>
 
 <partial name="_vacancyApprenticeshipPreview" model="@Model" />    
```

Comment:
Above this we will need the validation summary panel

Reply:
Amended in latest.